added functions start and stop, without tracker_id

dejazzhands opened the pull request:
- wrote the logic function start, query adds new event to the events table and set the start_time to the current time.
- wrote the logic functoin stop, query updates a currently running event and set the end_time to the current time.

### todo
- [ ] create routes in main.rs for said functions
- [ ] @samuel-skean review logic functions, whether or not we should assign tracker_id here or if we should do that in main.rs instead,

dejazzhands committed:
added functions start and stop, without tracker_id

samuel-skean commented:
The start and stop functions should take the tracker_id as parameters.
Are you intending the start/stop routes to return the created events? I'd rather only the stop route return an event, honestly, because I'd rather not reveal incomplete events (events without a stop time).
Please remove the empty migration files.

dejazzhands committed:
Delete migrations/migrations/20250207051648_20250124024534_Initial_tables..up.sql.sql

dejazzhands committed:
Delete migrations/migrations/20250207051702_20250124024534_Initial_tables..down.sql.sql

dejazzhands committed:
Delete migrations/migrations/20250207052334_20250124024534_Initial_tables..down.sql.sql

dejazzhands committed:
Delete migrations/migrations/20250207052342_20250124024534_Initial_tables..up.sql.sql

dejazzhands committed:
Delete migrations/migrations/20250207053223_20250124024534_Initial_tables..up.sql.sql

dejazzhands committed:
Update logic.rs

not returning an event for tracker_events_start

dejazzhands committed:
response to skean's comments:
1. added migrations content to gitignore
2. can't really add entries without returning the event
3. added tracker_id to scope in tracker_events so Ok wouldn't be a problem.

dejazzhands committed:
response to skean's comments:
1. No clean way to insert without returning event
2. added migrations/ and subfolders to .gitignore
3. Specified tracker_id inside Some so that it's properly scoped
4. Added tracker_id as parameters to event start and stop functions

dejazzhands committed:
Merge branch 'samef-changes' of github.com:samuel-skean/trackerman into samef-changes
